tests/common/connections/telnet_console_conn.py

```diff
@@ -0,0 +1,26 @@
+from base_console_conn import BaseConsoleConn
+
+class TelnetConsoleConn(BaseConsoleConn):
+    def __init__(self, **kwargs):
+        kwargs['device_type'] = "_telnet"
+        super(TelnetConsoleConn, self).__init__(**kwargs)
+
+    def session_preparation(self):
+        super(TelnetConsoleConn, self).session_preparation()
+
+    def telnet_login(
+        self,
+        pri_prompt_terminator=r".*# ",
+        alt_prompt_terminator=r".*\$ ",
+        username_pattern=r"(?:user:|username|login|user name)",
+        pwd_pattern=r"assword",
```

Comment:
Typo in `assword`?

Reply:
I think this is work work with leading 'P' and 'p'